frontend/packages/component-library/package.json

```diff
@@ -16,13 +16,15 @@
   },
   "license": "SEE LICENSE IN LICENSE",
   "sideEffects": false,
-  "type": "module",
   "exports": {
     "./package.json": "./package.json",
     "./divider": {
-      "types": "./dist/divider/index.d.ts",
-      "import": "./dist/divider/index.js",
-      "require": "./dist/divider/index.cjs"
+      "types": {
+        "import": "./dist/divider/index.d.mts",
```

Comment:
Do I understand correctly that we will need to follow this approach for all components? (https://github.com/code-dot-org/code-dot-org/pull/63395 - use same approach for all components here)

Reply:
Yes, this is correct. Up to you if you want to merge first and I can update this PR to reflect those changes.

Reply:
Just saw you already pushed and updated this!